cmd/libs/go2idl/deepcopy-gen/main.go

```diff
@@ -59,9 +59,7 @@ func main() {
 	arguments.OutputFileBaseName = "deep_copy_generated"
 
 	// Custom args.
-	customArgs := &generators.CustomArgs{
-		BoundingDirs: []string{"k8s.io/kubernetes"},
-	}
+	customArgs := &generators.CustomArgs{}
```

Comment:
trying to make the tool less kube-specific, maybe bust it into a separate repo eventually.  These are useful.